components/webhook-mgt/org.wso2.carbon.identity.webhook.management/src/main/java/org/wso2/carbon/identity/webhook/management/api/constant/ErrorMessage.java

```diff
@@ -45,9 +45,11 @@ public enum ErrorMessage {
     ERROR_INVALID_WEBHOOK_REQUEST_FIELD("WEBHOOKMGT-60012", "Invalid request.",
             "%s is invalid."),
     ERROR_CODE_WEBHOOK_ALREADY_ACTIVE("WEBHOOKMGT-60013", "Webhook already active",
-            "The webhook: %s is already active and cannot be activated again."),
+            "The webhook: %s is already active and request accepted."),
```

Comment:
Previous one is right here

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/6885/commits/fb575c056228c93d2221b4ea29c500a76d09311c